saleor/graphql/channel/mutations/channel_create.py

```diff
@@ -24,6 +30,20 @@ class StockSettingsInput(graphene.InputObjectType):
     )
 
 
+class OrderSettingsInput(graphene.InputObjectType):
```

Comment:
I don't think, that we should add `required=True` to all fields here. You can want to modify only the single field.